curriculum/challenges/english/25-front-end-development/quiz-javascript-strings/66edc31c44f1b9c1d5c5ebca.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+How can you access the second character of the string `JavaScript` using bracket notation?
 
 #### --distractors--
 
-Placeholder distractor 1
+`'JavaScript'.charAt(2)`
 
 ---
 
-Placeholder distractor 2
+`'JavaScript'.charAt(1)`
 
 ---
 
-Placeholder distractor 3
+`'JavaScript'[2]`
 
 #### --answer--
 
-Placeholder answer
+`'JavaScript'[1]`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which option demonstrates string interpolation?
 
 #### --distractors--
 
-Placeholder distractor 1
+`'Hello, ' + user + '!'`
 
 ---
 
-Placeholder distractor 2
+`'Hello, ${user}!'`
 
 ---
 
-Placeholder distractor 3
+`` `Hello, {user}!` ``
 
 #### --answer--
 
-Placeholder answer
+`` `Hello, ${user}!` ``
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which option creates a string with a newline character?
 
 #### --distractors--
 
-Placeholder distractor 1
+`'\t'`
 
 ---
 
-Placeholder distractor 2
+`'\b'`
 
 ---
 
-Placeholder distractor 3
+`'\p'`
 
 #### --answer--
 
-Placeholder answer
+`'\n'`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which option creates a string with a tab character?
 
 #### --distractors--
 
-Placeholder distractor 1
+`'\n'`
 
 ---
 
-Placeholder distractor 2
+`'\b'`
 
 ---
 
-Placeholder distractor 3
+`'\p'`
 
 #### --answer--
 
-Placeholder answer
+`'\t'`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which option creates a string object?
 
 #### --distractors--
 
-Placeholder distractor 1
+`String('Hello')`
 
 ---
 
-Placeholder distractor 2
+`'Hello'`
 
 ---
 
-Placeholder distractor 3
+`createString('Hello')`
 
 #### --answer--
 
-Placeholder answer
+`new String('Hello')`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which option extracts a substring consisting of the first three characters from the string `JavaScript`?
 
 #### --distractors--
 
-Placeholder distractor 1
+`'JavaScript'.substr(3)`
 
 ---
 
-Placeholder distractor 2
+`'JavaScript'.substr(0, 4)`
 
 ---
 
-Placeholder distractor 3
+`'JavaScript'.extract(0, 3)`
 
 #### --answer--
 
-Placeholder answer
+`'JavaScript'.substr(0, 3)`
```

Comment:
`.substr()` is deprecated. We should probably use `.substring()`.

Reply:
Thank you for pointing that out. I've completely forgot about the deprecation of `.substr()`. I will update the code to use `.substring()` instead.